AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -2488,6 +2488,8 @@ izlemac75.net##body > div#reload
 !
 ||yargiyayinevi.com/Data/EditorFiles/banners/eylul21/slider-alt
 bessarabiainform.com##.news-bar
+wionews.com#?#.article-main-data > p > strong:contains(WATCH)
+wionews.com##.article-main-data > p iframe[src^="https://www.wionews.com/live-tv"]
```

Comment:
It can be blocked by URL `https://www.wionews.com/live-tv/embed?autoplay=1&mute=1` to safe traffic.